.github/workflows/integration-tests.yml

```
@@ -0,0 +1,37 @@
+            .cpcache
+          key: ${{ runner.os }}-clojure-${{ hashFiles('**/deps.edn') }}
+          restore-keys: |
+            ${{ runner.os }}-clojure-
+
+      - name: Run integration tests
+        run: |
+          echo "Current directory: $(pwd)"
+          echo "Script path: ./test/scripts/integration-tests.sh"
+          chmod +x ./test/scripts/integration-tests.sh
+          ./test/scripts/integration-tests.sh
+        shell: /usr/bin/bash -e {0}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)